cheat_sheets/hashes.md

```diff
@@ -0,0 +1,20 @@
+---
+layout: cheat_sheet
+topic: Hashes
+---
+
+| Name               | Description                                                          | Syntax                                        | Examples                                                                             |
+| ------------------ | -------------------------------------------------------------------- | --------------------------------------------- | ------------------------------------------------------------------------------------ |
+| Hash literal       | Create a hash                                                        | _{ "key" => "value" }_                        | `hash = {}` <br> `hash = { "key" => "value", "other_key" => 42 }`                    |
+| Hash access        | Access a value by its key                                            | _hash[key]_                                   | `hash["key"]`                                                                        |
+| Key deletion       | Delete a key-value pair by its key                                   | _hash.delete(key)_                            | `hash.delete("key")`                                                                 |
+| Empty hash         | Remove all pairs from the hash                                       | _hash.clear_                                  | `hash.clear`                                                                         |
+| Iterate over hash  | Iterate over all the pairs in the hash                               | _hash.each do \|key, value\|_ <br> _end_      | `hash.each do |key, value|` <br> `puts "#{key} has value: #{value}"` <br> `end`      |
+| Iterate over pairs | Iterate over all the pairs in the hash                               | _hash.each_pair do \|key, value\|_ <br> _end_ | `hash.each_pair do |key, value|` <br> `puts "#{key} has value: #{value}"` <br> `end` |
```

Comment:
They didn't have Enumerables yet, just sayin'

Reply:
Yeah, i think i'll leave it in there still, for completeness...